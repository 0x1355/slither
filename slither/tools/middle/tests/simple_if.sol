pragma solidity >=0.4.16 <0.7.0;

contract SimpleIf {
    uint public a;

    function f(uint input) public {
        if(input == 2)
            a = 1;
        else
            a = 0;
    }
}