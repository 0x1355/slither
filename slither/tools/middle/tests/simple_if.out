OVERLAY_FUNCTION_
    ENTRY_POINT
    EXPRESSION
OVERLAY_FUNCTION_
    ENTRY_POINT
    EXPRESSION
f
    ENTRY_POINT
    IF
    OVERLAY
    OVERLAY
    END_IF
